src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -40,7 +45,32 @@
      * София - 900</p>
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException, ParseException {
+        final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
https://stackoverflow.com/questions/66066/what-is-the-best-way-to-implement-constants-in-java